helm/charts/oxtrust/values.yaml

```diff
@@ -0,0 +1,55 @@
+# Default values for oxtrust.
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
+gluuMaxRamFraction: 1
+
+dynamicStorage: true
+
+image:
+  repository: nginx
+  tag: stable
+  pullPolicy: Always
+
+service:
+  type: ClusterIP
+  port: 80
+
+# PVC storage
+volumeClaim:
+  storage: 10M
+
+#resources: 
+limits:
+  memory: 128Mi
+requests:
+  memory: 300Mi
+
+#containerPOrt ot connect the container with
+containerPort: 8080
+
+gluuOxauthBackend: oxauth:8080
+
+#service values
+#port for which to open in oxtrust ss
+port: 8080
+clusterIp: None
+
+#VolumeMounts
+volumeMounts:
+  shared-shibboleth-idp:
+    mountPath: /opt/shared-shibboleth-idp
+    name: shared-shib
```

Comment:
Multisharing volume on the cloud is not possible. Shared shib needs to stay locally and statically provisioned here.